src/function/getJournal.ts

```diff
@@ -14,7 +15,8 @@ export function getJournal(event: any, context: Context, callback: Callback) {
       response = createResponse({ message }, 500);
       callback(err, response);
     } else {
-      response = createResponse({ reply }, 200);
+      const { data: testSlots } = JSON.parse(JSON.parse(reply));
```

Comment:
how comes we need to parse this twice?

Reply:
some weirdness around a string within a string. A single JSON.parse returns a string, the double returns the actual data object. I thought removing the stringify on the set function might help, but didn't seem to affect it.